blog/introducing-testcafe-open-source-testing-framework.html

```diff
@@ -251,6 +251,27 @@
     <div class="page-content-wrapper">
 
       
+
+
+<script>
+    var TEAM_BLOG_LAST_VISITED_FLAG = 'team-blog-last-visited';
+    var TEAM_BLOG_LINK_ID           = 'blog-link';
+    var TEAM_BLOG_LINK_NEW_CLASS    = 'new-post';
+    var NOTIFICATIONS_ACTIVE_SINCE  = new Date(2020, 4, 21)
+
+    var teamBlogLastVisited = window.localStorage.getItem(TEAM_BLOG_LAST_VISITED_FLAG) || NOTIFICATIONS_ACTIVE_SINCE;
+
+    window.addEventListener('DOMContentLoaded', function (e) {
+        var lastPostDate = new Date(1590354000 * 1000);
```

Comment:
Better to use more readable format for this date and even extract it to a const

Reply:
It looks like that in code:

https://github.com/DevExpress/testcafe-gh-page-assets/blob/master/src/_includes/header.html#L65

You're looking at the complied site

Reply:
ok, sure